react/src/lib/components/EdsIcon/EdsIcon.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
is this import statement necessary? I don't see where you use `React` below.

Reply:
This seems to be a leftover from the copy-paste operation and can be removed ;)

Reply:
ah, yummy copypasta, my favorite ^^

Reply:
I stay corrected: before, each `JSX` component had to be in the React scope - hence, React must be imported, no matter if it is used or not. This is because the `JSX` file is converted to a JavaScript file in which the `React.createElement` is used (see here: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html). The new JSX transform does not require this anymore, but we don't have the setup adjusted yet. Hence, we have to keep it for now.

Reply:
aha, right! that makes sense, I read about the `JSX` and how it's transformed. cool! thanks for the further explanation ^^